NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift

```diff
@@ -671,6 +678,10 @@ extension AudiobookPlayerViewController: AudiobookNetworkServiceDelegate {
     public func audiobookNetworkService(_ audiobookNetworkService: AudiobookNetworkService, didReceive error: NSError?, for spineElement: SpineElement) {
         presentAlertAndLog(error: error)
         self.audiobookProgressView.stopShowingProgress()
+        if let error = error,
+          error.domain == OverdrivePlayerErrorDomain && error.code == 4 {
```

Comment:
can you put that magic number into a more descriptive constant? Perhaps/hopefully there's already an enum for collecting all framework-level errors somewhere?